code/modules/reagents/chemistry/reagents/other_reagents.dm

```diff
@@ -470,110 +470,124 @@
 	name = "Podperson Mutation Toxin"
 	id = "podmutationtoxin"
 	description = "A vegetalizing toxin."
-	color = "#5EFF3B" //RGB: 94, 255, 59
 	race = /datum/species/pod
 	mutationtext = "<span class='danger'>The pain subsides. You feel... plantlike.</span>"
 
 /datum/reagent/mutationtoxin/jelly
 	name = "Imperfect Mutation Toxin"
 	id = "jellymutationtoxin"
 	description = "An jellyfying toxin."
-	color = "#5EFF3B" //RGB: 94, 255, 59
 	race = /datum/species/jelly
 	mutationtext = "<span class='danger'>The pain subsides. You feel... wobbly.</span>"
 
 /datum/reagent/mutationtoxin/golem
 	name = "Golem Mutation Toxin"
 	id = "golemmutationtoxin"
 	description = "A crystal toxin."
-	color = "#5EFF3B" //RGB: 94, 255, 59
 	race = /datum/species/golem/random
 	mutationtext = "<span class='danger'>The pain subsides. You feel... rocky.</span>"
 
 /datum/reagent/mutationtoxin/abductor
 	name = "Abductor Mutation Toxin"
 	id = "abductormutationtoxin"
 	description = "An alien toxin."
-	color = "#5EFF3B" //RGB: 94, 255, 59
 	race = /datum/species/abductor
 	mutationtext = "<span class='danger'>The pain subsides. You feel... alien.</span>"
 
 /datum/reagent/mutationtoxin/android
 	name = "Android Mutation Toxin"
 	id = "androidmutationtoxin"
 	description = "A robotic toxin."
-	color = "#5EFF3B" //RGB: 94, 255, 59
 	race = /datum/species/android
 	mutationtext = "<span class='danger'>The pain subsides. You feel... artificial.</span>"
 
+/datum/reagent/mutationtoxin/fungus
+	name = "Fungus Mutation Toxin"
+	id = "fungusmutationtoxin"
+	description = "A fungus toxin."
+	race = /datum/species/mush
+	mutationtext = "<span class='danger'>The pain subsides. You feel... shroomy.</span>"
+
 
 //BLACKLISTED RACES
 /datum/reagent/mutationtoxin/skeleton
 	name = "Skeleton Mutation Toxin"
 	id = "skeletonmutationtoxin"
 	description = "A scary toxin."
-	color = "#5EFF3B" //RGB: 94, 255, 59
 	race = /datum/species/skeleton
 	mutationtext = "<span class='danger'>The pain subsides. You feel... spooky.</span>"
 
 /datum/reagent/mutationtoxin/zombie
 	name = "Zombie Mutation Toxin"
 	id = "zombiemutationtoxin"
 	description = "An undead toxin."
-	color = "#5EFF3B" //RGB: 94, 255, 59
 	race = /datum/species/zombie //Not the infectious kind. The days of xenobio zombie outbreaks are long past.
 	mutationtext = "<span class='danger'>The pain subsides. You feel... undead.</span>"
 
 /datum/reagent/mutationtoxin/ash
 	name = "Ash Mutation Toxin"
 	id = "ashmutationtoxin"
 	description = "An ashen toxin."
-	color = "#5EFF3B" //RGB: 94, 255, 59
 	race = /datum/species/lizard/ashwalker
 	mutationtext = "<span class='danger'>The pain subsides. You feel... savage.</span>"
 
+/datum/reagent/mutationtoxin/angel
+	name = "Angelic Mutation Toxin"
+	id = "angelicmutationtoxin"
+	description = "A holy toxin."
+	race = /datum/species/angel
+	mutationtext = "<span class='danger'>The pain subsides. You feel... angelic.</span>"
+
+/datum/reagent/mutationtoxin/headless
```

Comment:
Dullahans are bug central and only kept in the codebase because they're only available during halloween or when an admin is online to deal with the mess